File: existDB-backend/src/main/java/hu/sule/administration/model/FileManagerEntity.java
package hu.sule.administration.model;


import lombok.*;

@AllArgsConstructor
@NoArgsConstructor
@Data
public class FileManagerEntity {
    private boolean isFile;
    private String name;
    private String size;
    private String humanSize;
    private String modified;
    private boolean hidden;
    private boolean canRead;
    private boolean canWrite;
}

File: existDB-frontend/src/app/file-manager/model/FileManagerEntity.ts
export class FileManagerEntity {
    isFile: boolean;
    name: string;
    size: string;
    humanSize: string;
    modified: string;
    hidden: boolean;
    canRead: boolean;
    canWrite: boolean;

    constructor(isFile: boolean, name: string, size: string, humanSize: string, modified: string, hidden: boolean, canRead: boolean, canWrite: boolean) {
        this.isFile = isFile;
        this.name = name;
        this.size = size;
        this.humanSize = humanSize;
        this.modified = modified;
        this.hidden = hidden;
        this.canRead = canRead;
        this.canWrite = canWrite;
    }
}
